docs/reference/4-command-line-tools/1-goliothctl/goliothctl_stream_listen.md

```diff
@@ -0,0 +1,47 @@
+---
+id: goliothctl_stream_listen
+title: "goliothctl stream listen"
+slug: goliothctl_stream_listen
+sidebar_label: goliothctl stream listen
+url: /reference/command-line-tools/goliothctl/goliothctl_stream_listen/
+hide_title: true
+---
+## goliothctl stream listen
+
+Listen to stream messages from all devices of the project or filter them by device using the device name, device id or the hardware id.
+
+```bash
+goliothctl stream listen [device name] [flags]
+```
+
+### Examples
+
+```bash
+# Listen to stream messages from all devices of the project
+> goliothctl stream listen
+
+# Listen to stream messages from a specific device 
+> goliothctl stream listen <device name>
+> goliothctl stream listen --id <device id>
+> goliothctl stream listen --hwId <hardware id>
+```
+
+### Options
+
+```bash
+  -h, --help               help for stream listen
```

Comment:
Please confirm that I have used the correct help text here.

Reply:
I don't think it's a problem Mike.